src/main/resources/hudson/security/LDAPSecurityRealm/config.jelly

```diff
@@ -60,7 +60,7 @@ THE SOFTWARE.
         <f:entry field="disableMailAddressResolver">
             <f:checkbox title="${%Disable Ldap Email Resolver}"/>
         </f:entry>
-        <f:entry field="disableRolePrefixing" help="/plugin/ldap/help-disableRolePrefixing.html">
+        <f:entry field="disableRolePrefixing">
```

Comment:
help file is https://github.com/jenkinsci/ldap-plugin/blob/c02a69c9787f5da0957757f8c47054f14b720c13/src/main/resources/hudson/security/LDAPSecurityRealm/help-disableRolePrefixing.html#L4